quarkus-extensions-with-dependency/src/main/java/io/quarkus/qe/ExcelUtils.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.qe;
+
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Font;
+
+
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+public class ExcelUtils {
+
+    private ExcelUtils(){}
+
+    public static void writeToExcel(String searchedDependency, List<Extension> extensions) throws IOException {
+        try (Workbook workbook = new XSSFWorkbook()) {
+            Sheet sheet = workbook.createSheet(
+                    String.format("Quarkus extensions with %s dependency", searchedDependency));
+
+            // Create header row
+            Row headerRow = sheet.createRow(0);
+            // Bold text for header row cells
+            CellStyle headerCellStyle = workbook.createCellStyle();
+            Font headerFont = workbook.createFont();
+            headerFont.setBold(true);
+            headerCellStyle.setFont(headerFont);
+
+            Cell nameCell = headerRow.createCell(0);
+            nameCell.setCellValue("Name");
+            nameCell.setCellStyle(headerCellStyle);
+
+            Cell descriptionCell = headerRow.createCell(1);
+            descriptionCell.setCellValue("Build with Quarkus");
+            descriptionCell.setCellStyle(headerCellStyle);
+
+            Cell artifactCell = headerRow.createCell(2);
+            artifactCell.setCellValue("Artifact");
+            artifactCell.setCellStyle(headerCellStyle);
+
+            // Create data rows
+            int rowNum = 1;
+            for (Extension extension : extensions) {
+                Row row = sheet.createRow(rowNum++);
+                row.createCell(0).setCellValue(extension.getName());
+                row.createCell(1).setCellValue(extension.getMetadata().getBuiltWithQuarkusCore());
+                row.createCell(2).setCellValue(extension.getArtifact());
+            }
+
+            // Auto-size columns based on the widest content in each column
+            for (int i = 0; i < headerRow.getLastCellNum(); i++) {
+                sheet.autoSizeColumn(i);
+            }
+
+            // Write to Excel file
+            try (FileOutputStream fileOut = new FileOutputStream("extensions.xlsx")) {
```

Comment:
The file extensions.xlsx hardcoded could be defined as a parameter to make it configurable.

Reply:
Now you can configure custom file name 
`./get-extensions.sh quarkus-vertx [custom-myfile.xlsx]`

Reply:
ok, good change but let's wait to see your new changes with another more open format way that has been suggested, thanks.